donkeycar/parts/web_controller/web.py

```diff
@@ -365,10 +367,16 @@ async def get(self):
         while True:
 
             interval = .01
-            if served_image_timestamp + interval < time.time() and \
-                    hasattr(self.application, 'img_arr'):
```

Comment:
I believe you are doing this, because two different web servers are using the VideoAPI. Wouldn't it be better to make `img_arr` a member in both of them?

Reply:
I don't remember doing anything here.  Let me look into it.  I don't know what this is supposed to be doing, but it is also a delete.  So this must have come from a rebase  somehow.

Reply:
Ok, I see.  I changed the code to show a placeholder image if not camera image is available.  The full code change is:
```
            if served_image_timestamp + interval < time.time():
                #
                # if we have an image, then use it.
                # otherwise show placeholder
                #
                if hasattr(self.application, 'img_arr') and self.application.img_arr is not None:
                    img = utils.arr_to_binary(self.application.img_arr)
                else:
                    img = utils.arr_to_binary(placeholder_image)
```